src/components/tabs/js/tabsController.js

```diff
@@ -395,7 +395,7 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
           isActive:     function () { return this.getIndex() === ctrl.selectedIndex; },
           isLeft:       function () { return this.getIndex() < ctrl.selectedIndex; },
           isRight:      function () { return this.getIndex() > ctrl.selectedIndex; },
-          shouldRender: function () { return !ctrl.noDisconnect || this.isActive(); },
+          shouldRender: function () { return ctrl.noDisconnect || this.isActive(); },
```

Comment:
I'm wondering, why there is another attribute `md-no-disconnect`, which disabled the `disconnect`functionality.

So that means, that we have two attributes (which are both reversed)
- `md-enable-disconnect` (Documented)
- `md-no-disconnect` (Not documented) 

@robertmesserle Do you know anything about that?